feature-configs/demo/sriov/11-sriov-networknodepolicy.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-network-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true
+  mtu: 1500
+  nicSelector:
+    pfNames:
+    - eno1
```

Comment:
I would use this as base and use kustomize to replace the pf name of the given environment.
This will make stuff easier once we have a new envorinment.

Also, only kustomization should stay under 'demo'.